examples/rendercrds.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+
+# Latest semver release does not compile, gg controller-tools.
+# Override it with this for now.
+go install sigs.k8s.io/controller-tools/cmd/controller-gen
+
+mkdir -p tmp
```

Comment:
what about $(mktemp -d) so this works cleanly if the script is ^C 1/2 way throught?

Reply:
`$()` is a bash-ism :)